pkg/osquery/table/launcher_info.go

```diff
@@ -57,7 +58,7 @@ func generateLauncherInfoTable(configStore types.GetterSetter, LauncherHistorySt
 			return nil, err
 		}
 
-		osqueryInstance, err := history.LatestInstance()
+		osqueryInstance, err := history.LatestInstanceByRegistrationID(types.DefaultRegistrationID)
```

Comment:
I don't think we want to return multiple rows for this table -- I'm not sure how K2 will handle that at the moment. So for now, I'm defaulting to using default ID (the same as we do in other places).

Reply:
You can return whatever makes sense here, multiple rows are fine.

This isn't data k2 collects, it's here solely for our debugging.

Reply:
Correcting myself... We _do_ query `kolide_launcher_info`, but I don't think we look at this column.